lib/routes/telegram/tglib/client.ts

```diff
@@ -24,73 +24,7 @@ if (config.telegram.session) {
     });
```

Comment:
https://github.com/DIYgod/RSSHub/blob/94a78a70bf72463460c3ff17d7b2872f9236981f/lib/routes/telegram/tglib/client.ts#L22

Do not throw plain string, throw `new Error(contain)` instead.